gatsby-config.js

```diff
@@ -32,5 +32,6 @@ module.exports = {
     `gatsby-plugin-sharp`,
     `gatsby-transformer-remark`,
     `gatsby-plugin-sass`,
+    `gatsby-plugin-react-helmet`
```

Comment:
It's weird that this is being added again. It was added in your last PR. I suppose we'll see what happens 🤔 

https://github.com/swfl-coders/website/commit/f25d6f213d67123171927c1e7aeb3573f2ee7f1c#diff-b5e305780d9d473da97c61beab8bc36e5e8871b360942e4686c9b20d8c5d4cfaR35

Reply:
yes i did not know how to get rid of it from this PR after sending it over earlier. Is there any way to remove it from this PR? or are we just going to see if anything happens?